contracts/proofs/ExitValidator.sol

```diff
@@ -66,7 +71,7 @@ contract ExitValidator is RootChainValidator {
     address owner;
     uint256 amount;
     bool burnt;
-    (owner,,amount, burnt) = withdrawManager.getExit(exitId);
+    (owner,,amount, burnt) = IWithdrawManager(registry.getWithdrawManagerAddress()).getExit(exitId);
```

Comment:
Maybe we should pass `_withdrawManager` in constructor. That's why we made it upgradeable.

Reply:
what's correlation between passing _withdrawManager and `we made it upgradeable`?

Reply:
We are accessing vars from registry
1. that can change
2. Are being used at more than 1 place.
Since WM is upgradeable, we could keep it in constructor. anyway not clear what would be better so ok for now.